eng/Version.Details.xml

```diff
@@ -35,37 +35,37 @@
     <Dependency Name="Microsoft.WindowsAppSDK" Version="0.0.1">
       <Uri>https://dev.azure.com/microsoft/ProjectReunion/_git/ProjectReunionInternal</Uri>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.Authorization" Version="6.0.4">
-      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-aspnetcore</Uri>
-      <Sha>f9ae0f5d30be2de3c0de61b5673bd8873231d70a</Sha>
+    <Dependency Name="Microsoft.AspNetCore.Authorization" Version="6.0.5">
```

Comment:
Are these packs going to ship the same timeframe as the 6.0.5 dotnet/runtime?

Just checking that these will be on NuGet.org when MAUI ships. /cc @Redth as a reviewer

Reply:
> Are these packs going to ship the same timeframe as the 6.0.5 dotnet/runtime?

It was going to be in a very tight timeframe. I think the decision we'd reached was we want to hold off on merging this PR (and thereby upgrading from 6.0.4 -> 6.0.5) till after GA to be safe.

> Mainly wondering if this PR should be merged by someone who has the right rights…
@dougbu 

No it should _NOT_ be merged. Ideally we'd only consume this package update once the packages are publicly available, but unfortunately, that doesn't [seem to be possible](https://github.com/dotnet/maui/pull/6081#issuecomment-1098463264). So blocking the PR was the alternative.

Reply:
@TanayParikh the workflow above will get you what you want, once the build is officially released (week from Tuesday).

Reply:
Any change here? i.e. is this PR able to be merged?